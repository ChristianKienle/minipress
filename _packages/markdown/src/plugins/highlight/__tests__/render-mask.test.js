// @ts-check
const render = require('./../render-mask')

describe('highlight markdown plugin â€“ render mask', () => {
  it('works', () => {
    const result = render({
      highlightLines: [1,2,3],
      escapedContent:
`export default {
 hello: 'world'
}`})
    expect(result).toMatchSnapshot()
  })
})
