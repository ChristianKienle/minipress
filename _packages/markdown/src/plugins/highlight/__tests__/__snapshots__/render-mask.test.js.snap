// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`highlight markdown plugin â€“ render mask works 1`] = `"<div class=\\"minipress-highlight-mask\\"><div class=\\"code-line highlighted\\">export default {</div><div class=\\"code-line highlighted\\"> hello: 'world'</div><div class=\\"code-line highlighted\\">}</div></div>"`;
